components/create-pull-request-modal.tsx

```diff
@@ -76,26 +78,32 @@ export default function CreatePullRequestModal({
       const filteredRepos = 
         repositories.filter(repo => (repo.isFork && repo.nameWithOwner === `${user.login}/${activeRepo?.name}`) 
                                     || repo.nameWithOwner === activeRepo?.githubPath);
-
       
-
       return Promise.all(filteredRepos
         .map(async (repository) => ({ repository, branches:  await getRepositoryBranches(repository.nameWithOwner)})));
     })
     .then(reposWithBranches => reposWithBranches
-      .map(({ repository, branches }) => branches.map(branch => ({ 
-        value: `${repository.owner}:${branch}`, 
-        label: branch,
-        postIcon: <Badge 
-                    color={repository.isOrganization ? "white-10" : "primary-30"}
-                    label={repository.isOrganization ? t("misc.organization") : t("misc.fork")}
-                  />,
-        isSelected: !!selectedBranch && branch === selectedBranch
-      })))
+      .map(({ repository, branches }) => branches
+        .map(branch => { 
+          const isDisabled =  !!activeIssue.pullRequests.find(pr=> pr.branch.split(':')[1] === branch)
```

Comment:
We need to verify branch and repo, because user can open a pull request using a branch of a cloned repo or an organization repo.

Ex.: `vhcsilva/web-network#dev` and `bepro-organization/web-network#dev`, the user should be able to open a PR for `dev` branch of `vhcsilva/web-network` and other PR for `bepro-organization/web-network` .

Do you agree @moshmage ?